### PR TITLE
Adds chainId and yParity; removes senderV

### PR DESCRIPTION
**Pull request by MicahZoltu**
`chainId` was missing from abstract.

Discussion in Discord landed on switching over to `yParity` instead of `v` since `chainId` so now part of the payload explicitly and the `+27` stuff is tech debt from Bitcoin.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 2930 requires approval from one of (@holiman, @vbuterin)

File: EIPS/eip-2930.md
Comment (holiman) [inline EIPS/eip-2930.md:36]: Now you changed it back so `0` means wildcard. Why?

Comment (MicahZoltu) [reply EIPS/eip-2930.md]: Good catch, copy/paste error (copied from 2711, where `0` cross-chain transactions are allowed).  Fixed now.

File: EIPS/eip-2930.md
Comment (holiman) [inline EIPS/eip-2930.md:36]: Nitpick, it currently sounds like it doesn't actually _become_ valid until it's included in a block. Or even: until the `value` is included in a block
Maybe:
```suggestion
**`ChainId`** The transaction only valid on networks with this `chainID`.
```

File: EIPS/eip-2930.md
Comment (holiman) [inline EIPS/eip-2930.md:49]:
```suggestion
As of `FORK_BLOCK_NUMBER`, we introduce a new transaction type, with the format `1 || rlp([chainId, nonce, gasPrice, gasLimit, to, value, data, access_list, yParity, senderR, senderS])`. The signing hash would be the hash of the RLP of the entire data structure but popping the last three items representing the signature, and prepending the transaction type: `rlp([1, chainId, nonce, gasPrice, gasLimit, to, value, data, access_list])`.
```
You name it `YParity` above, not `SenderYParity`

Comment (holiman) [reply EIPS/eip-2930.md:49]: Well, either one is fine, but better be consistent

**Review by holiman — APPROVED** [review]

File: EIPS/eip-2930.md
Comment (MicahZoltu) [reply EIPS/eip-2930.md]: I put the `sender` prefix to align with `senderR` and `senderS`, where `r` and `s` and `yParity` are the three outputs of the `secp256k1` signing algorithm.  That being said, I'm fine with your suggestion as well and have applied it.

File: EIPS/eip-2930.md
Comment (MicahZoltu) [reply EIPS/eip-2930.md]: Applied, I like your wording better than mine.  😄

File: EIPS/eip-2930.md
Comment (holiman) [inline EIPS/eip-2930.md:19]:
```suggestion
We introduce a new [EIP-2718](./eip-2718.md) transaction type, with the format `1 || rlp([chainId, nonce, gasPrice, gasLimit, to, value, data, access_list, yParity, senderR, senderS])`.
```
Sorry, one more :)

Comment (MicahZoltu) [reply EIPS/eip-2930.md]: Luckily CI is running slow so I'm able to get these applied before CI runs and the auto-merge bot merges (since you already approved).  😬

**Pull request merged**
